data_utils/raw_dataset_loader.py

```diff
@@ -101,7 +111,7 @@ def __transform_input_to_data_object_base(
             for icomp in range(graph_feature_dim[item]):
                 it_comp = graph_feature_col[item] + icomp
                 g_feature.append(float(graph_feat[it_comp].strip()))
-        data_object.y = tensor([g_feature])
```

Comment:
Is this a bug? Or just cleaner this way?

Reply:
> Is this a bug? Or just cleaner this way?

Yeah, it is a typo.